cluster_test.go

```diff
@@ -24,54 +24,107 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestUrl(t *testing.T) {
-	cluster, _ := newCluster(http.DefaultClient, fakeMarathonURL)
-	assert.Equal(t, cluster.URL(), fakeMarathonURL)
-}
-
 func TestSize(t *testing.T) {
 	cluster, _ := newCluster(http.DefaultClient, fakeMarathonURL)
```

Comment:
Not sure why we are ignoring the error return value here and in many other related tests. I'd probably just `assert.NotNil` on the value.